deploy/openshift/templates/che-server-template.yaml

```diff
@@ -153,6 +153,8 @@ objects:
             value: "${CHE_WORKSPACE_PLUGIN__REGISTRY__URL}"
           - name: CHE_TRACING_ENABLED
             value: "${CHE_TRACING_ENABLED}"
+          - name: CHE_CORS_ALLOW_CREDENTIALS
```

Comment:
Have you checked it? For me, it looks wrong. additional `_` is missing after `ALLOW`